backend/accounts.go

```diff
@@ -931,6 +931,7 @@ func (backend *Backend) uninitAccounts() {
 			backend.onAccountUninit(account)
 		}
 		account.Close()
+		backend.emitAccountsListChanged()
```

Comment:
at this stage `backend.accounts` has not changed at all though, so why fire an event here?